question24/main.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+const agreementLinkBtn = document.getElementById('js-agreement-link');
```

Comment:
Q
There is a button in the variable name,
I think it's actually a text link to the Terms of Service, but what about it?

Reply:
Thank you for your review! It is true..! I have tried changing the variable names, but what do you think? Please confirm.

fixed commit : https://github.com/csakyo/homework/pull/38/commits/fbe563af8d1a66168477e091060273c3c69dfcd0